### PR TITLE
Make presto-mongodb tests actually parallel

### PR DESCRIPTION
**Pull request by ArturGajowy**
This shaves off 2 min (of 13) from the longest Travis job
and 6 min (of 11) on my machine.

### CODE REVIEW

**Comment by ArturGajowy** [comment]
Together with #5471 this will further reduce build result waiting time by 2 mins.

**Comment by ArturGajowy** [comment]
I don't know why, but Github/Travis didn't trigger a PR build for this. Branch build here: https://travis-ci.org/Teradata/presto/builds/138831579.

**Comment by losipiuk** [comment]
Doesn't it break anything?
I guess that thread count was set to 1 for a reason here.

**Comment by ArturGajowy** [comment]
I ran the tests locally and they passed. So they did at Travis. I think it's just a copy-paste from cassandra connector.

**Comment by losipiuk** [comment]
You may try to ping author of mongodb connector on that. But otherwise it looks fine to me.

File: presto-mongodb/pom.xml [hunk 1]
```diff
@@ -189,7 +189,6 @@
                 <configuration>
                     <!-- integration tests take a very long time so only run them in the CI server -->
                     <parallel />
-                    <threadCount>1</threadCount>
```
Comment (ArturGajowy) [inline presto-mongodb/pom.xml]: Hi @miniway! @losipiuk suggested that this line might actually be needed. Do you remember putting it on purpose? Were there any test failures when running in parallel? Please see the discussin in #5472.

File: presto-mongodb/pom.xml [hunk 2]
```diff
@@ -189,7 +189,6 @@
                 <configuration>
                     <!-- integration tests take a very long time so only run them in the CI server -->
                     <parallel />
```
Comment (electrum) [inline presto-mongodb/pom.xml]: Remove `<parallel />` as well, so it uses the default

**Comment by martint** [comment]
@electrum, do you have any further comments on this?

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**